### PR TITLE
New vms 2017

### PR DESCRIPTION
**Pull request by vext01**
This is to accompany the VMs update.

Mostly updated support for TruffleRuby, but some doc fixes too.

### CODE REVIEW

**Comment by vext01** [comment]
Don't merge until the other PR is in.

Tinkering with the idea of squashing these PRs into one easily revertible commit. Thoughts?

File: krun/vm_defs.py [hunk 1]
```diff
@@ -631,37 +631,27 @@ def __init__(self, vm_path, env=None):
...
         """
         Args:
-            jruby_dir: path to a (built) jruby src dir
+            truffleruby_dir: path to a (built) truffleruby src dir
             graal_home: path to the built graal-core directory
             mx_dir: path to mx directory
 
-        Note that this is for the open-source graal-core powered JRbuy, not
-        for the proprietary graalvm binaries found on Oracle Technology Network.
+        Note that this is for the open-source graal-core powered TruffleRbuy,
```
Comment (snim2) [inline krun/vm_defs.py]: TruffleRuby misspelled at end of line

**Comment by snim2** [comment]
Other than that typo, looks OK to me

**Comment by snim2** [comment]
Want to squash?

**Comment by vext01** [comment]
Yes, into one easily revertible commit. Is that OK?

**Comment by snim2** [comment]
Yeah, but I'd keep the detail in the commit messages

**Comment by snim2** [comment]
OK, ready for merging?

**Comment by vext01** [comment]
Let's get the other PR ready and merge at the same time

**Pull request merged**